src/components/MySongs/MySongs.js

```diff
@@ -54,11 +56,27 @@ export class MySongsComponent {
     }
   }
 
+  handleRemoveSong(songId) {
+    MusicService.removeUserSong(AuthService.getCurrentUser().uid, songId).then(
+      () => this.mount()
+    );
+  }
+
+  addSong() {
+    if (this.mountPoint.querySelector(".my-songs")) {
```

Comment:
move to queryselectors

Reply:
This way doesn't work in the case when this component is mount at least once